packages/main/src/Select.hbs

```diff
@@ -1,23 +1,35 @@
 <div
 	class="ui5-select-root"
-	tabindex="{{tabIndex}}"
 	dir="{{dir}}"
 	@keydown="{{_onkeydown}}"
 	@keyup="{{_onkeyup}}"
 	@focusin="{{_onfocusin}}"
 	@focusout="{{_onfocusout}}"
 	@click="{{_toggleRespPopover}}"
 >
-	<div class="ui5-select-label-root">
-		<ui5-label>{{_text}}</ui5-label>
-	</div>
+	<div class="ui5-select-wrapper"
```

Comment:
Is it really necessary to have one more nested `div` with `width: 100%; height: 100%`?

Reply:
Actually, no. This was needed in order to wrap the label and the icon together and add all relevant aria attributes, when the popover was part of the ui5-select. Now this is not needed. 

@nnaydenow you could add aria attributes from the ui5-select-wrapper ui5-select-root. Sorry for not checking that earlier.

Reply:
Done